README.md

```diff
@@ -82,6 +82,26 @@ Neo4j::Http::Client.find_nodes_by(label: "User", name: "Testing")
 Neo4j::Http::Client.delete_node(node)
 ```
 
+### Upsert and delete nodes in batch via UNWIND
```

Comment:
I presume delete would be similar except `Neo4j::Http::Client.delete_node(node, unwind: [...]`)